src/config.rs

```diff
@@ -975,6 +981,86 @@ impl Config {
             (value - (number_of_crs_to_reserve as i32))..value,
         ))
     }
+
+    pub(crate) fn can_read(
+        &self,
+        req: &actix_web::HttpRequest,
+        document_path: &str,
+    ) -> fpm::Result<bool> {
+        use itertools::Itertools;
+        let access_identities = {
+            if let Some(identity) = req.cookie("identities") {
+                fpm::user_group::parse_identities(identity.value())
+            } else {
+                self.local_identities.clone()
+            }
+        };
+
+        let id = self
+            .doc_id()
+            .unwrap_or_else(|| document_path.to_string())
```

Comment:
Rename this to document_name. Use variable names as per glossary as far as possible.